shared/common-adapters/form-input.native.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import * as React from 'react'
+import {Box, Text} from '.'
```

Comment:
we shouldn't import from . inside common adapters. creates import loops and in general we've been avoiding it. just import ./box and ./text